eradiate/data/absorption_spectra.py

```diff
@@ -48,7 +134,7 @@ def open(cls, id):
             raise OSError(f"while opening file at {path}: {str(e)}")
 
     @classmethod
-    def find(cls):
+    def find(cls, test=False):
```

Comment:
This the `test` argument used somewhere?

Reply:
No, that argument is a leftover from a debug session, I forgot to remove it.